src/plot/axis/axis-title.js

```diff
@@ -36,30 +36,23 @@ const propTypes = {
 function AxisTitle({orientation, width, height, title}) {
   const x = orientation === LEFT ? width : 0;
   const y = orientation === TOP ? height : 0;
-  let style;
+  let transform;
+  let textAnchor;
   if (orientation === LEFT) {
-    style = {
-      transform: 'translate(1em, 0) rotate(-90deg)',
-      textAnchor: 'end'
-    };
+    transform = 'translate(1em, 0) rotate(-90deg)';
+    textAnchor = 'end';
   } else if (orientation === RIGHT) {
-    style = {
-      transform: `translate(-0.5em, ${height}px) rotate(-90deg)`,
-      textAnchor: 'start'
-    };
+    transform = `translate(-0.5em, ${height}px) rotate(-90deg)`;
+    textAnchor = 'start';
   } else if (orientation === BOTTOM) {
-    style = {
-      transform: `translate(${width}px, -6px)`,
-      textAnchor: 'end'
-    };
+    transform = `translate(${width}px, -6px)`;
+    textAnchor = 'end';
   } else {
-    style = {
-      transform: 'translate(0px, 1em)'
-    };
+    transform = 'translate(0px, 1em)';
```

Comment:
does this work on edge? my understanding of svg transforms was that was done in unitless values? I'm fiddling around with my own version and this technique throws a lot of errors